src/test/java/com/github/princesslana/eriscasper/faker/DataFaker.java

```diff
@@ -47,6 +54,38 @@ public static User user() {
         .build();
   }
 
+  public static Guild guild() {
+    Snowflake id = DiscordFaker.snowflake();
+    return ImmutableGuild.builder()
+        .id(id)
+        .name(Faker.instance().name().name())
+        .ownerId(DiscordFaker.snowflake())
+        .region(Faker.instance().address().country())
+        .afkTimeout(0L)
+        .verificationLevel(0L)
+        .defaultMessageNotifications(0L)
+        .explicitContentFilter(0L)
+        .mfaLevel(0L)
+        .addChannels(guildChannel(id))
+        .build();
+  }
+
+  public static Channel channel() {
```

Comment:
I'd prefer to have one faker per resource, with individual tests changing the values if needed. So, either `channel` or `guildChannel` (but renamed to `channel`), but not both.